applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -30,6 +34,31 @@ class ResponseFunctionBase(object):
     solving of primal (and adjoint) analysis ...
     """
 
+    def __init___(model_part, response_settings):
```

Comment:
missing self?

Reply:
yep, was not called, thats why i did not notice... removed it